CMakeLists.txt

```diff
@@ -31,33 +36,67 @@ elseif (WIN32)
 endif ()
 
 # Will be overwritten during the cmake initialization.
+
+# Value can be one of "None", "ControlFlow-GNU", ""ControlFlow-Clang" and "ControlFlow" (which is alias to ControlFlow-GNU)
+# If None, no LVI mitigation will be applied
+# If ControlFlow, only control-flow mitigations will be applied
+# If ControlFlow-Clang, the Clang's built-in mitigation will be applied (only support clang-11 and later)
+# If ControlFlow-GNU, the custom mitigation will be applied
+# Support of full-LVI hardening (-mlvi-hardening) is omitted
 set(LVI_MITIGATION
     "None"
-    CACHE STRING "Build with LVI mitigation.")
+    CACHE
+      STRING
+      "Build with LVI mitigation. Options: [None, ControlFlow-GNU, ControlFlow-Clang, ControlFlow]"
+)
 set(LVI_MITIGATION_BINDIR
     "None"
     CACHE STRING "Path to the LVI mitigation bindir.")
 
+# alias ControlFlow to ControlFlow-GNU
+if (LVI_MITIGATION STREQUAL "ControlFlow")
+  set(LVI_MITIGATION "ControlFlow-GNU")
+endif ()
```

Comment:
@CyanDevs  - Which one do we test in CI?

Reply:
We test both ControlFlow and ControlFlow-Clang, but the majority of tests are ran with ControlFlow for now.